tests/unit_tests/data_model/test_validate_data.py

```diff
@@ -3,6 +3,8 @@
 import logging
 import sys
 
+import astropy.io.registry
```

Comment:
Here you did import just a submodule, but perhaps could even import a smaller module

Reply:
Note that I am still catching `astropy.io.registry.base.IORegistryError`, so this is not consistent. I've changed it now consistently to 'astropy' (I get errors when importing the smallest module (not sure if this is related to 'base'). I think this is really a style issue and leave it as it is.